### ISSUE TITLE
[FEAT]: Support configuring Dependabot on GitHub Actions runners

### ISSUE THREAD

**Issue by stevehipwell** [integrations/terraform-provider-github#2250]
### Describe the need

I'd like to be able to configure [Dependabot on GitHub Actions runners](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/about-dependabot-on-github-actions-runners). Based on the announcement this would be required at the organisation and repo levels.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nnellanspdl** [comment]
Would love to see these options implemented as well.

**Comment by risset** [comment]
It seems like this setting can't be set through the GitHub API

**Comment by Okeanos** [comment]
Would still love to see these options implemented.

**Comment by gabrielsoltz** [comment]
+1 Would love to see these options implemented!

**Comment by eterhoven** [comment]
+1 Would love to see these options implemented!

**Comment by Okeanos** [comment]
Would still like to see this implemented.

**Comment by stevehipwell** [comment]
I don't think this issue is relevant any more as I opened it specifically to deal with the migration of Dependabot to runners, which IIRC has been completed now.
